src/Uri.php

```diff
@@ -221,7 +225,7 @@ private function __construct(
      */
     private function formatScheme(?string $scheme): ?string
     {
-        if (null === $scheme) {
+        if (null === $scheme || in_array($scheme, ['http', 'https', 'ws', 'wss'], true)) {
```

Comment:
after merging I have changed this line to 
```
        if (null === $scheme || isset(self::SCHEME_DEFAULT_PORT[$scheme]))
```

Reply:
👍